Cargo.toml

```diff
@@ -55,6 +55,7 @@ rstest = "0.18"
 sspi = "0.11"
 tracing = { version = "0.1", features = ["log"] }
 thiserror = "1.0"
+png = "0.17"
```

Comment:
Thought: besides the local crates, I would prefer we only put "ubiquitous" dependencies in the workspace manifest. Following this logic, we would probably only keep `tracing` and `thiserror`. Since all tests should go into the testsuite crate, it’s likely `proptest`, `rstest` and `expect-test` don’t belong here either.

Reply:
Well, on the other hand, globally defined dependency versions allow very easy version upgrades and reduce the possibility of duplicated libraries (via human error)

Reply:
This is true, but we use don't upgrade manually anymore (dependabot does that for us)